db/README.md

```diff
@@ -9,75 +9,70 @@ Detailed setup documentation that was followed can be found [here](https://docs.
 #### The following 5 tables are entity tables.
 
 ### user
-| Column      | Data Type | Detail |
-| ----------  | --------- | -------|
-| `user_id` | serial | Primary Key, auto-incremented |
-| `oauth_token` | varchar(255) | Generated by OAuth |
-| `email` | varchar(255) | Email of user |
-| `user_type` | varchar(255) | Values = {cleaner, employer} |
-| `is_verified` | boolean | Default: false |
+| Column      | Data Type |  Description | Detail |
+| ----------  | --------- | ------------ | ------ |
+| `user_id` | serial | Auto-incremented ID starting at 1 | Primary Key, Not Null |
+| `auth_id` | varchar(25) | Generated by OAuth | Not Null |
+| `user_type` | enum | Values = {cleaner, employer} | Not Null |
+| `is_verified` | boolean | | Default: false | Not Null, Default: 0 |
 
 ### certification
-| Column      | Data Type | Detail |
-| ----------  | --------- | -------|
-| `cert_id` | serial | Primary Key, auto-incremented |
-| `cert_name` | varchar(255) | Name of the certification |
-| `cert_category` | varchar(255) | e.g. Window Cleaning, Mopping, etc |
-| `is_verified` | boolean | Default: false |
+| Column      | Data Type |  Description | Detail |
+| ----------  | --------- | ------------ | ------ |
+| `cert_id` | serial  | Auto-incremented ID starting at 1 | Primary Key, Not Null |
+| `cert_name` | varchar(255) | Name of the certification | Not Null |
 
 ### cleaner_profile
-| Column      | Data Type | Detail |
-| ----------  | --------- | -------|
-| `profile_id` | serial | Primary Key, auto-incremented |
-| `first_name` | varchar(255) | First name of cleaner |
-| `last_name` | varchar(255) | Last name of cleaner |
-| `contact_num` | varchar(50) | Contact number of cleaner e.g. '4161234567' |
-| `contact_ext` | varchar(10) | Default: NULL, not mandatory |
+| Column      | Data Type |  Description | Detail |
+| ----------  | --------- | ------------ | ------ |
+| `profile_id` | serial | Auto-incremented ID starting at 1 | Primary Key, Not Null |
```

Comment:
Is this also a FK  to the user_id generated by Auth0?

Reply:
The id generated by Auth0 will be stored as 'auth_id', and it's not a FK to any table
auth_id is 1 to 1 associated with user_id (internal AoC ID) and that user_id will be associated with profile_id via the has_profile relationship table, where user_id and profile_id are both FK!